libs/custom-paper-handler/src/driver/driver.ts

```diff
@@ -547,61 +628,59 @@ export class PaperHandlerDriver {
   ): Promise<USBOutTransferResult> {
     assertNumberIsInRangeInclusive(numMotionUnits, 0, 1600);
     const [nH, nL] = Uint16toUint8(numMotionUnits);
-    return this.transferOutGeneric([...SET_PRINTING_AREA_WIDTH, nL, nH]);
+    return this.transferOutGeneric(SetPrintingAreaWidthCommand, { nL, nH });
   }
 
   async setLineSpacing(numMotionUnits: Uint8): Promise<USBOutTransferResult> {
-    return this.transferOutGeneric([...SET_LINE_SPACING, numMotionUnits]);
+    return this.transferOutGeneric(SetLineSpacingCommand, { numMotionUnits });
   }
 
   async setPrintingSpeed(
     printingSpeed: PrintingSpeed
   ): Promise<USBOutTransferResult> {
-    return this.transferOutGeneric([
-      ...SET_PRINTING_SPEED,
-      PRINTING_SPEED_CODES[printingSpeed],
-    ]);
+    return this.transferOutGeneric(SetPrintingSpeedCommand, {
+      speed: PRINTING_SPEED_CODES[printingSpeed],
+    });
   }
 
   async setPrintingDensity(
     printingDensity: PrintingDensity
   ): Promise<USBOutTransferResult> {
-    return this.transferOutGeneric([
-      ...SET_PRINTING_DENSITY,
-      PRINTING_DENSITY_CODES[printingDensity],
-    ]);
+    return this.transferOutGeneric(SetPrintingDensityCommand, {
+      density: PRINTING_DENSITY_CODES[printingDensity],
+    });
   }
 
+  // TODO why do these functions accept as input a Uint16 instead of 2 Uint8s?
```

Comment:
It's setting only the horizontal units, not the vertical units, so there's only one axis / value. So a single number makes more sense than 2 numbers.